lib/taskbuilder.js

```diff
@@ -375,7 +389,30 @@ exports.build = function (ops) {
         groupItems;
 
     if (ops.command === "group") {
-        return buildGroup(ops);
+        var groupItems;
+
+        // get the group data from the json file
+        try {
+            groupItems = require(GROUPS_DIR + "/" + ops.commandValue + ".json");
+        } catch (e) {
+            return Promise.reject("Couldn't find group with name " + ops.commandValue);
+        }
+
+        return buildGroup(groupItems, ops);
+
+    } else if (ops.command === "build-group") {
+        return childProcess.execAsync(GROUP_BUILDER_DIR + "/" + ops.commandValue)
```

Comment:
Why did you have to fork out to a Perl script? Was it not possible to use the same metadata JSON that you use for the `ia` command?